nncf/experimental/torch/sparsify_activations/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nncf.experimental.torch.sparsify_activations.sparsify_activations_impl import sparsify_activations  # noqa: F401
```

Comment:
@nikita-savelyevv Thank you for the suggestion.

We wonder if it will be more appropriate to place the documentation at [nncf/docs/usage/post_training_compression](https://github.com/openvinotoolkit/nncf/tree/develop/docs/usage/post_training_compression) with a new folder `activation_sparsity`. Our concern is that a readme at this folder may be a little hidden. Or should we do both? any thoughts?

Reply:
@vuiseng9 In any case we will expose the link to the activations sparsification documentation at our main README file so it will be visible from there. For example this is how it's done for Movement Sparsity method [here](https://github.com/openvinotoolkit/nncf/tree/develop?tab=readme-ov-file#training-time-compression-algorithms). 

Actually, I think this should be done in this PR. We should add a new row for activation sparsity algorithm to the Post-Training Compression Algorithms [table](https://github.com/openvinotoolkit/nncf/tree/develop?tab=readme-ov-file#post-training-compression-algorithms) and state experimental support for PyTorch backend. cc @yujiepan-work 

@vuiseng9 Will this address your concerns regarding the exposure of the algorithm documentation?

Reply:
@nikita-savelyevv yes, that is a better idea! 

We will include the documentation in this PR. Thanks.